tests/integration/test_hpc_libs.yaml

```diff
@@ -38,11 +38,21 @@ acts:
           venv/bin/python3 -m pip install -r dev-requirements.txt
       - name: "Run `slurm_ops` integration tests"
         run: |
+          PYTHONPATH=./lib \
           venv/bin/python3 -m pytest -v \
             -s \
             --tb native \
             --log-cli-level=INFO \
             slurm_ops
+      - name: "Run `slurm_ops` integration tests (snap)"
```

Comment:
GitHub won't let me comment on line 39, but could we do the same thing with the act name we're doing here by having the act name for the deb version of the integration test be:

```yaml
- name: "Run `slurm_ops` integration tests (deb)"
```

That way folks can easily tell what the differences between the tests are in the integration test logs :smiley: